database/store.go

```diff
@@ -35,6 +35,10 @@ type Store interface {
 	// version is not found, this method must return [ErrVersionNotFound].
 	GetMigration(ctx context.Context, db DBTxConn, version int64) (*GetMigrationResult, error)
 
+	// GetLatestVersion retrieves a latest version. If the query succeeds, but the
+	// version is not found, this method must return [ErrVersionNotFound].
```

Comment:
I don't think this comment is true, nothing returns `ErrVersionNotFound` from our underlying implementations.

Reply:
Really, I'd forgotten about that
[3867034](https://github.com/pressly/goose/pull/675/commits/3867034e6e652b28cc74e584980ce22e62814c64)